LondonBritishLibrary/orient/BLorient11188.xml

```diff
@@ -0,0 +1,290 @@
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 11188</idno>
+                        <altIdentifier><idno>Strelcyn 33</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="108r"/>
+                            <title ref="LIT6489SchoolChants"/>
+                            <notatedMusic><desc>Extensive musical notation.</desc></notatedMusic>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="3r" to="29r"/>
+                                <title ref="LIT2509Weddas/"></title>
+                                <note>Arranged for the days of the week with the reading for Monday being first.</note>
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="3r"/>
+                                    <title ref="LIT2509Weddas#Monday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.2">
+                                    <locus from="5v"/>
+                                    <title ref="LIT2509Weddas#Tuesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.3">
+                                    <locus from="10r"/>
+                                    <title ref="LIT2509Weddas#Wednesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.4">
+                                    <locus from="14v"/>
+                                    <title ref="LIT2509Weddas#Thursday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.5">
+                                    <locus from="20r"/>
+                                    <title ref="LIT2509Weddas#Friday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.6">
+                                    <locus from="23v"/>
+                                    <title ref="LIT2509Weddas#Saturday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.7">
+                                    <locus from="26v"/>
+                                    <title ref="LIT2509Weddas#Sunday"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="29v" to="42r"/>
+                                <title ref="LIT5646Mastagabe"/>
+                                <note>Sung on the <foreign xml:lang="gez">ʿǝzl</foreign> and the <foreign xml:lang="gez">gǝʿǝz</foreign> modes 
```

Comment:
I think the keywords for the modes instead of the the element foreign could be used here.

Reply:
You mean, I shall add key to title? What is the correct key to be introduced for Geez and Ezl?

```
<msItem xml:id="ms_i1.2.1">
<title ref="LIT5646Mastagabe#Monday"/>
<msItem xml:id="ms_i1.2.1.1">
<locus from="29v"/>
<title key="ezl"/>
</msItem>
<msItem xml:id="ms_i1.2.1.2">
<locus from="30r"/>
<title key="geez"/></title>
</msItem>
</msItem>
```

Reply:
I meant  `<note>Sung in the <term key="EzelMode">ʿǝzl</term > and the <term key="GeezMode">gǝʿǝz</term > modes ...`
(https://betamasaheft.eu/Guidelines/?id=term)

Reply:
I will change the title of the msItem only. The terms in the note need to stay like they are, correct?